services/ui-src/src/components/fields/ChoiceField.tsx

```diff
@@ -1,51 +1,86 @@
 import { useFormContext } from "react-hook-form";
 // components
 import { Choice as CmsdsChoice } from "@cmsgov/design-system";
-import { Box } from "@chakra-ui/react";
+import { Box, Text } from "@chakra-ui/react";
 // utils
 import { makeMediaQueryClasses } from "utils";
 import { InputChangeEvent, AnyObject } from "types";
+import { useEffect, useState } from "react";
 
-export const ChoiceField = ({
-  name,
-  type,
-  value,
-  label,
-  sxOverride,
-  ...props
-}: Props) => {
+export const ChoiceField = ({ name, label, sxOverride, ...props }: Props) => {
   const mqClasses = makeMediaQueryClasses();
+  const singleBoxClass = "single-box";
```

Comment:
Request: Actually, I'm not even sure we need this class, since it doesn't seem to be doing anything. I'm guessing this is an artifact from `CheckboxField`, but since that no longer needs to handle single checkboxes, we should change several lines of code from `CheckboxField` as part of this PR as well:

`CheckboxField`
- delete [line 16](https://github.com/CMSgov/mdct-mcr/blob/de5a0165b1a63f147c951a647adce859a2ac3447/services/ui-src/src/components/fields/CheckboxField.tsx#L16)
- modify [lines 20-21](https://github.com/CMSgov/mdct-mcr/blob/de5a0165b1a63f147c951a647adce859a2ac3447/services/ui-src/src/components/fields/CheckboxField.tsx#L20-L21)
- delete [lines 34-45](https://github.com/CMSgov/mdct-mcr/blob/de5a0165b1a63f147c951a647adce859a2ac3447/services/ui-src/src/components/fields/CheckboxField.tsx#L34-L45)